msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md

```diff
@@ -204,33 +204,15 @@ The language supports features that allow you to include variables, call externa
 
 ## Actions
 
-Actions handle events triggered by AI components.
-
-`FlaggedInputAction` and `FlaggedOutputAction` are the built-in action handlers to handle the moderator flags. If the moderator flags an incoming message input, the moderator redirects to the `FlaggedInputAction` handler and the `context.sendActivity` sends a message to the user about the flag.
-
-[Sample code reference](https://github.com/microsoft/teams-ai/blob/main/js/samples/04.ai.a.teamsChefBot/src/index.ts#L97)
-
-```javascript
-app.ai.action(AI.FlaggedInputActionName, async (context: TurnContext, state: TurnState, data: TData) => {
-    await context.sendActivity(`I'm sorry your message was flagged: ${JSON.stringify(data)}`);
-    return false;
-});
-
-app.ai.action(AI.FlaggedOutputActionName, async (context: TurnContext, state: ApplicationTurnState, data: TData) => {
-    await context.sendActivity(`I'm not allowed to talk about such things.`);
-    return false;
-});
-```
+Actions are atomic functions that are registered to the AI Module and serve as the fundamental building blocks of a plan. Actions allow you to define the specific tasks your app will perform and are pivotal in executing plans and enhancing your application's functionality. For instance, you might want your app to create a list, send a message, or perform a calculation. By creating and registering actions, you can control these behaviors and enhance your app's functionality.
 
 ### Register Action Handlers
 
-Action handlers help users achieve the goals, which is shared in the user intents.
-
-One of the key aspects in action handlers is that you must first register the actions in the prompts and then help user achieve the goal.
+Action handlers help users achieve the goals, which is shared in the user intents. One of the key aspects in action handlers is that you must first register the actions in the prompts and then help user achieve the goal.
 
-You must register a handler for each action listed in the prompt and also add a handler to deal with unknown actions.
+You must register a handler for each action listed in the prompt and also add a handler to deal with unknown actions. You can use the `app.ai.action` method to register an action.
 
-In the following example of a light bot, we have the `LightsOn`, `LightsOff`, and `Pause`  action. Every time an action is called, you return `true` or `false`. ​Returning `false` from a handler prevents the planner from running additional `DO` or `SAY` commands. When the bot receives an unknown action, we're telling the bot to terminate the action.
+In the following example of a light bot, we have the `LightsOn`, `LightsOff`, and `Pause`  actions. Every time an action is called, you return `true` or `false`. ​Returning `false` from a handler prevents the planner from running additional `DO` or `SAY` commands. When the bot receives an unknown action, we're telling the bot to terminate the action.
```

Comment:
In the given light bot example, the actions `LightsOn`, `LightsOff`, and `Pause` are present. Each time an action is invoked, it returns either `true` or `false`. If a handler returns `false`, it stops the planner from executing further `DO` or `SAY` commands. If the bot encounters an unknown action, it is instructed to terminate that action.

Reply:
updated.